app/views/request/_after_actions.html.erb

```diff
@@ -5,7 +5,7 @@
     <strong><%= _('Anyone:') %></strong>
 
     <ul>
-      <% if @info_request.comments_allowed? %>
+      <% if AlaveteliConfiguration.enable_annotations && @info_request.comments_allowed? %>
```

Comment:
You could bake `AlaveteliConfiguration.enable_annotations` in to `InfoRequest#comments_allowed?`.